src/main/java/org/eclipse/microprofile/starter/addon/microprofile/servers/model/StandaloneMPSpec.java

```diff
@@ -0,0 +1,106 @@
+/*
+ * Copyright (c) 2020 Contributors to the Eclipse Foundation
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * You may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+package org.eclipse.microprofile.starter.addon.microprofile.servers.model;
+
+import java.util.Arrays;
+import java.util.List;
+
+public enum StandaloneMPSpec {
+    // @formatter:off
+    GRAPH_QL("graphql", "Graph QL",
+            "https://github.com/eclipse/microprofile-graphql/releases/tag/%s",
+            "Graph QL: GraphQL is a query language for APIs and a runtime for fulfilling those queries with your existing data.",
```

Comment:
a runtime =>  a library

Reply:
Would it be possible to change the textual references to "GraphQL" (without the space)?  In the code, the `GRAPH_QL` with the underscore is fine, but in the user-facing text, it should be "GraphQL" to match both the specification from Facebook and the Microprofile project name.